hw/ip_templates/clkmgr/data/clkmgr.hjson.tpl

```diff
@@ -429,13 +428,14 @@
           name: "VAL",
           desc: '''
             Enable jittery clock.
-            A value of kMultiBitBool4False disables the jittery clock,
-            while all other values enable jittery clock.
+            At reset, this register reads as kMultiBitBool4False and the jittery clock is disabled.
+            Any write to the register turns the value to kMultiBitBool4True and enables the jittery clock.
+            The value written doesn't matter.
+            The value then remains kMultiBitBool4True until reset.
           ''',
           resval: false
-          // avoid writing random values to this register as it could trigger transient checks
-          // in mubi sync
-          tags: ["excl:CsrAllTests:CsrExclWrite"]
+          // Exclude this register from any automated CSR tests as the behavior is non-standard.
+          tags: ["excl:CsrAllTests:CsrExclAll"]
```

Comment:
Not a big deal, but is that exclusion change necessary? Which CSR test was failing?

Reply:
It turns out this is really required. Without the change, all CSR tests except for the csr_hw_reset test fail.

Reply:
This CSR is declared RW for software, so automated tests will expect after a write you will read back what you wrote. I expect the cip_lib csr_rw family of tests to fail.